examples/createwithcustomfields/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"syscall"
+
+	jira "github.com/andygrunwald/go-jira"
+	"github.com/trivago/tgo/tcontainer"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+func main() {
+	r := bufio.NewReader(os.Stdin)
+
+	fmt.Print("Jira URL: ")
+	jiraURL, _ := r.ReadString('\n')
+
+	fmt.Print("Jira Username: ")
+	username, _ := r.ReadString('\n')
+
+	fmt.Print("Jira Password: ")
+	bytePassword, _ := terminal.ReadPassword(int(syscall.Stdin))
```

Comment:
Can you update this to use the new [golang.org/x/term.ReadPassword](https://pkg.go.dev/golang.org/x/term#ReadPassword) which is part of `golang.org/x/term` instead since `golang.org/x/crypto/ssh/terminal` was deprecated? This new implementation seems to add support for windows as well which the previous version did not.